models/intel/text-recognition-0016/README.md

```diff
@@ -0,0 +1,85 @@
+# text-recognition-0016 (composite)
+
+## Use Case and High-Level Description
+
+This is a text-recognition composite model that recognizes scene text.
+The model uses predefined set of alphanumeric symbols (case-insensitive) to predict words.
+The model is built on the ResNeXt-101 backbone with [TPS](https://arxiv.org/abs/1603.03915) module and additional 2d attention-based text recognition head.
+
+## Example of the input data
+
+![](./assets/text-recognition-0016.jpg)
+
+## Example of the output
+
+`openvino`
+
+## Composite model specification
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy on the alphanumeric subset of ICDAR13 | 0.9685             |
+| Accuracy on the alphanumeric subset of ICDAR03 | 0.9712             |
+| Accuracy on the alphanumeric subset of ICDAR15 | 0.8675             |
+| Accuracy on the alphanumeric subset of SVT     | 0.9474             |
+| Accuracy on the alphanumeric subset of IIIT5K  | 0.9347             |
+| Text location requirements                     | Tight aligned crop |
+| Source framework                               | PyTorch\*          |
```

Comment:
probably Source Framework should be placed before accuracy numbers as it is more important and general info